pytest_pootle/fixtures/import_export_fixtures.py

```diff
@@ -65,6 +65,9 @@ def _cleanup():
 @pytest.fixture
 def en_tutorial_ts(settings, english_tutorial, ts_directory):
     """Require the en/tutorial/tutorial.ts store."""
+
+    english_tutorial.project.localfiletype = "ts"
+    english_tutorial.project.save()
```

Comment:
I think it is better to instead use a different project instead of hijacking tutorial.